packages/shared/src/graphql/squads.ts

```diff
@@ -40,6 +40,14 @@ export type CreateSquadOutput = {
   createSquad: Squad;
 };
 
+export const DELETE_SQUAD_MUTATION = gql`
```

Comment:
Can you also update the squadMembers here since @idoshamun implemented it.

Reply:
I'll add a separate task for it as for deletion it doesn't make a difference.